### PR TITLE
Pass angles for SweepGradient in degrees, not radians

### PR DESCRIPTION
**Pull request by hterkelsen**
## Description

Fixes SweepGradient in CanvasKit mode by passing the angles in degrees, not radians.

## Related Issues

Related to: https://github.com/flutter/flutter/issues/41389

Here is how that repro looks now:
Screen Shot 2020-10-07 at 3 21 49 PM

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [contributor guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [C++, Objective-C, Java style guides] for the engine.
- [x] I read the [tree hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation.
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[contributor guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[tree hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by ferhatb — COMMENTED** [review]
Please add test.

**Comment by yjbanov** [comment]
Let's add a test that runs on Chrome (iOS Simulator, which takes Safari screenshots for us, can't run CanvasKit yet).

File: lib/web_ui/test/canvaskit/sweep_gradient_golden_test.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      final CkPaint paint = CkPaint()..shader = gradient;
+
+      canvas.drawRect(region, paint);
+
+      await matchPictureGolden(
+          'canvaskit_sweep_gradient.png', recorder.endRecording(),
+          write: true);
+    });
+  });
```
Comment (yjbanov) [inline lib/web_ui/test/canvaskit/sweep_gradient_golden_test.dart:74]: This probably needs `skip: isIosSafari || isFirefox`

**Review by yjbanov — APPROVED** [review]
lgtm

File: lib/web_ui/test/canvaskit/sweep_gradient_golden_test.dart (hunk 1 above)
Comment (hterkelsen) [reply lib/web_ui/test/canvaskit/sweep_gradient_golden_test.dart]: Done

**Pull request merged**